pkg/host-disk/host-disk_test.go

```diff
@@ -106,17 +80,23 @@ var _ = Describe("HostDisk", func() {
 		hostDiskCreatorWithReserve = NewHostDiskCreator(recorder, 10, 1048576, root)
 
 	})
+	createHostDiskPath := func(volumeName string) string {
+		err := os.Mkdir(path.Join(tempDir, volumeName), 0755)
+		if !os.IsExist(err) {
+			Expect(err).NotTo(HaveOccurred())
+		}
```

Comment:
I would like the following better:

```
if err != nil {
    Expect(err).ToMatchError(os.IsExist, "IsExist")
}
``

Reply:
Also, I would keep the old name (or change to `createHostDisk`) and add the capacity to this helper.

Reply:
what about 
```go
createHostDisk := func(volumeName string) string {
if err := os.Mkdir(path.Join(tempDir, volumeName), 0755); err != nil {
Expect(err).To(MatchError(os.ErrExist))	
}
return path.Join(tempDir, volumeName, "disk.img")
}
```